labonneboite/importer/jobs/geocode.py

```diff
@@ -34,11 +34,19 @@
 session = requests.session()
 session.mount('http://', adapter)
 jobs = []
-pool = Pool(pool_size)
 
 CITY_NAMES = load_city_codes()
 GEOCODING_STATS = {}
 
+manager = Manager()
+#Shared variables between processes
+adresses_not_geolocated = manager.list() #list to store the geolocations not saved in cache
+coordinates_updates = manager.list() #list to store the coordinates to update
```

Comment:
TIL `multiprocessing.Manager()` \o/ you're the boss @JimmyDore

Reply:
Vraiment très pratique haha